packages/kas/src/nodes.js

```diff
@@ -493,39 +513,47 @@ _.extend(Expr.prototype, {
         expr.hints = _.clone(this.hints);
         expr.hints[hint] = true;
         return expr;
-    },
+    }
 
-    hints: {
-        parens: false,
-    },
+    // hints = {
+    //     parens: false,
+    // };
```

Comment:
This can be deleted.

Reply:
Yes, I'll delete it's before landing.

Reply:
I checked and it's removed in the TS PR.  Once I land both of them, this'll go away.